### DISCUSSION TITLE
GAUGE HEAT FLUX GAS change in default behavior FDS 6.8.0->6.9

### DISCUSSION THREAD

**Discussion by johodges** [firemodels/fds#12573] [General]
FDS 6.8.0 lets the user define a device for GAUGE HEAT FLUX GAS without specifying an orientation. The latest build sends an error message requiring an ORIENTATION be specified, which I agree is a good change. What was the default behavior before if orientation was not specified? Should this be mentioned in the release notes?

**Comment by mcgratta** [comment]
I suppose. I forget what the default was. Probably (0,0,-1). I consider this a bug fix, and since we make hundreds of them between releases, we don't list in the release notes. But I'll add.

**Comment by johodges** [comment]
No problem. This came up helping a colleague troubleshoot a numerical instability in a complex model for 6.8. I wanted to see if it still crashes on the latest source and I had to add the ORIENTATIONS. My thought was: "if it came up for me once already and I was having trouble identifying where the change was, we might as well put it in a discussion topic at least so it pops up in a future search."
